### DISCUSSION TITLE
xorg library deprecation warnings from linuxbrew

### DISCUSSION THREAD

**Discussion by mellery451** [Homebrew/discussions#52] [Everyday usage] [answered]
These warnings started showing up a week or two ago whenever I try to update or install:

```
Warning: Use libfontenc instead of deprecated linuxbrew/xorg/libfontenc
Warning: Use libice instead of deprecated linuxbrew/xorg/libice
Warning: Use libpthread-stubs instead of deprecated linuxbrew/xorg/libpthread-stubs
Warning: Use libsm instead of deprecated linuxbrew/xorg/libsm
Warning: Use libx11 instead of deprecated linuxbrew/xorg/libx11
Warning: Use libxau instead of deprecated linuxbrew/xorg/libxau
Warning: Use libxext instead of deprecated linuxbrew/xorg/libxext
Warning: Use libxfixes instead of deprecated linuxbrew/xorg/libxfixes
Warning: Use libxt instead of deprecated linuxbrew/xorg/libxt
Warning: Use xorgproto instead of deprecated linuxbrew/xorg/xorgproto
```

Attempting to uninstall shows lots of other dependent formulae. What's the deal with these warnings and should I do something to resolve them?

**Comment by gromgit** [comment, marked as answer]
I think `brew untap linuxbrew/xorg` should be sufficient. All the listed formulae are now in the core tap, hence the warnings.

**Reply by mellery451** [reply]
that seems to do the trick - thanks!

**Comment by gabn88** [comment]
I have the same problem. But I get an error when I do brew untap linuxbrew/xorg. Error: Refusing to untap linuxbrew/xorg because it contains the following installed formulae or casks:
libomxil-bellagio
